File: backend/src/003-archivos/tipos/json/manage_json.rb
#!/usr/bin/ruby

# Importamos el módulo json de Ruby
require "json"

# Parsear el contenido de un archivo JSON
archivo = File.read('fichero.json')

puts archivo["py"]  # py

# --------------------------------

# Parsear cadena de texto JSON
texto = '{"hola": "que tal"}'
puts texto.class  #  String

response = JSON.parse(texto)
puts response.class  #  Hash

puts response["hola"]  # que tal

File: backend/src/003-archivos/tipos/json/manage_json.js
"use strict"

// Abrir un archivo JSON (NodeJS)
var fs = require('fs');
var archivo = JSON.parse(fs.readFileSync('fichero.json', 'utf8'));

console.log(archivo.py)  // 178

// --------------------------------

// Parsear cadenas de texto JSON
var texto = '{"hola": "que tal"}'
console.log(typeof(texto));  //  string

var response = JSON.parse(texto); // <---- ¡Igual que en Ruby!
console.log(typeof(response));  //  object

console.log(response.hola);  //  que tal
